schema/core/reusable/atomic_data/numeric_value.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Nit: If we call it `value_number` and `value_string` it would be easier to find in the file tree and more factual